docs/plugins/repository-s3.asciidoc

```diff
@@ -447,3 +447,9 @@ limitations.
 
 Instances residing in a public subnet in an AWS VPC will connect to S3 via the
 VPC's internet gateway and not be bandwidth limited by the VPC's NAT instance.
+
+Note that Amazon has published instructions at 
+https://docs.aws.amazon.com/cli/latest/reference/ec2/create-vpc-endpoint.html
+that permit you to create a gateway endpoint that will expose access to S3 to 
+endpoints within subnets within your VPC, which will ameliorate the network 
```

Comment:
is this correct? According to [S3 pricing](https://aws.amazon.com/s3/pricing/) `Data transferred out to an Amazon Elastic Compute Cloud (Amazon EC2) instance, when the instance is in the same AWS Region as the S3 bucket.`. I think the goal of a VPC endpoint is to avoid needing a public IP for accessing S3. But I might be wrong.

Reply:
We were fairly certain that we had a cost reduction when we switched to using an endpoint, and certainly the traffic on the NAT was reduced (of course).

Perhaps to be more prudent, could remove the statement about using the VPC endpoint as a cost reduction method, unless it's substantiated by more than just my anecdotal evidence?